spec/factories/requests.rb

```diff
@@ -41,4 +47,8 @@ def random_request_items
   trait :pending do
     status { 'pending' }
   end
+
+  trait :with_duplicates do
+    request_items { request_items_w_duplicates }
```

Comment:
I think it would actually be more straightforward to inline the function here rather than define it up above.

Reply:
I've got no problem with that.  Commented it more thoroughly in the process.